src/deadline/houdini_submitter/python/deadline_cloud_for_houdini/submitter.py

```diff
@@ -48,19 +48,24 @@ def _get_houdini_version() -> str:
 def _get_scene_asset_references(rop_node: hou.Node) -> AssetReferences:
     # collect input filenames
     asset_references = AssetReferences()
-    input_filenames: set[str] = set()
-    input_filenames.add(_get_hip_file())
+    asset_references.input_filenames.add(_get_hip_file())
+
     for parm, ref in hou.fileReferences():
-        if parm:
-            if parm.node() == rop_node:
-                continue
-            if ref.startswith(IGNORE_REF_VALUES):
-                continue
-            if parm.name() in IGNORE_REF_PARMS:
-                continue
-        if os.path.isdir(ref):
+        if not parm:
```

Comment:
nit: Can this be cleaned up a bit either through conjoining some of the conditions or changing to a if->elif->else so it's not so much of a wall of if->continues. If that makes things less readable though we can also leave it as-is.

Reply:
I'll take a look.